apis/management.cattle.io/v3/rke_types.go

```diff
@@ -229,6 +229,8 @@ type BaseService struct {
 	ExtraArgs map[string]string `yaml:"extra_args" json:"extraArgs,omitempty"`
 	// Extra binds added to the nodes
 	ExtraBinds []string `yaml:"extra_binds" json:"extraBinds,omitempty"`
+	// Extra environment variables added to the services
+	ExtraEnv []string `yaml:"extra_env" json:"extraEnv,omitempty"`
```

Comment:
@superseb could we elaborate on how those env vars are gonna be used. @galal-hussein suggested `this is to provide extra env variable to the docker container running kubernetes service`